teams/archive/docs-peers.toml

```diff
@@ -6,6 +6,9 @@ leads = []
 members = []
 alumni = [
     "alercah",
+    "Havvy",
+    "marioidival",
+    "matthewjasper",
```

Comment:
This archived team was made in #1134 to have somewhere to move alercah out of the old alumni.toml. [These are the rest of the members](https://github.com/rust-lang/team/commit/2e0e7821670c242249d39c3d84ecef063c11b3cc#diff-eefe8b65d03b2469336eb5da9474cd071c3e5f094105f4808ef2584b5489033c) (without the one who does not already have a profile in the people directory).